### DISCUSSION TITLE
how to increase the speed of address mode?

### DISCUSSION THREAD

**Discussion by fre2mansur** [albertobsd/keyhunt#337] [General]
I'm running address mode for Puzzle 67 with 8 GB RAM currently my speed is ~4 Mkeys/s (4115253 keys/s)   
is there any way that i could increase the speed to Tera/s or Peta/s?

**Comment by albertobsd** [comment]
No for rmd/address there is no way to boost the speed, that RAM speed boost is only for Publickeys with BSGS method.
